data/trainers/parties.asm

```diff
@@ -5187,6 +5194,15 @@ endc
 	dbp 61, WIGGLYTUFF
 	db -1 ; end
 
+	; LASS
+	db "Abigail@"
+	db TRAINERTYPE_NORMAL
+	; party
+	dbp LEVEL_FROM_BADGES + 3, SKIPLOOM
+	dbp LEVEL_FROM_BADGES + 4, FLAAFFY
+	dbp LEVEL_FROM_BADGES + 5, GOLDUCK
+	db -1 ; end
```

Comment:
Just a note: this will conflict with FIQ's PR #1141 ... So it just depends on who merges first, the other will need to fix :P.